src/components/icon/Dashboard.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import Icon from './Icon.js';
+
+export default function Dashboard({ active, ...props }) {
+  return (
+    <Icon viewBox="0 0 80 80" {...props}>
+      {active ? (
```

Comment:
seems like the only thing changing is the color. I think we should just set the right color based on the `active` value, this will prevent duplicating the code

Reply:
I guess that is technically more efficient since the vectors don't change.